src/localization/es/learn.json

```diff
@@ -0,0 +1,163 @@
+{
+  "examples.starter.title": "Práctica",
+  "examples.starter.description": "Ahora es el momento de hacer práctica algo para consolidar lo que hemos aprendido.",
+
+  "examples.finish.title": "¡Practica completada!",
+  "examples.finish.description": " ",
+
+  "steps.starter.title": "Intro",
+  "steps.starter.description": "RegEx es la abreviación de `Regular Expression`. Ayudar a coincidir, encontrar o gestionar el texto. Comience escribiendo `OK` en el campo Regex para pasar al primer paso y acceder a la descripción más detallada.",
```

Comment:
`Ayudar a coincidir, encontrar o gestionar el texto.` 

It should look like the following. But I am not sure. 🔽 

`Ayuda a hacer coincidir, buscar o administrar texto.`